demo/project_demo04-SecOps/run.py

```diff
@@ -14,8 +14,10 @@
 SCRIPT_DIR = os.path.join(os.path.dirname(__file__))
 
 
-def prepare():
-    cmd = ["cargo", "run", "--release"]
+def prepare(args=[2000000]):
+    assert len(args) == 1, "Expected one '--prepare-args' argument for num_pipelines"
+    num_pipelines = args[0]
+    cmd = ["cargo", "run", "--release", "--", "%s" % num_pipelines]
```

Comment:
can `--release` use os.environ["RUST_BUILD_PROFILE"] (see below) otherwise this might download stuff?